Yosemite/Yosemite/Model/Enums/CardPresentPaymentsOnboardingState.swift

```diff
@@ -9,8 +9,10 @@ public enum CardPresentPaymentOnboardingState: Equatable {
     case completed(plugin: CardPresentPaymentsPluginState)
 
     /// There is more than one plugin installed and activated. The user must deactivate one.
+    /// `pluginSelectionWasCleared` being true means that there was one plugin selected for payments
+    /// but that selection was just cleared (e.g when in settings asking to choose a plugin again). Being `nil` means that we do not have that information.
```

Comment:
Does `false` have a meaning? It looks like it isn't actually possible.

Perhaps another enum for this would be more expressive? It might just add overhead though, I'm not sure. Perhaps using `false` instead of nil and making it non-optional would be better...

Reply:
Yeah, I wanted to avoid setting it to `false` in the case when we do not have that knowledge. But you are right, we can simplify it by making it non optional 😅 Done in https://github.com/woocommerce/woocommerce-ios/pull/7172/commits/a51f4e2acc4a2a824e3a8e58a392f9ae2d18e8a7